bin/mpdevil

```diff
@@ -3360,9 +3359,11 @@ class AudioFormat(Gtk.Box):
 		self._client=client
 		self._settings=settings
 		self._top_label=Gtk.Label(xalign=1)
-		self._brate_label=Gtk.Label(xalign=1, width_chars=5)
```

Comment:
I recently added this extra label to ensure the bitrate always fits in the given space. When using just one label to display the complete top line in AudioFormat the calculation of width_chars to reserve enough space is not accurate.

Reply:
Can the character reservation be done by _top_label instead?

Reply:
Yes sure, but it is inaccurate since numbers are usually wider than some other chars like the slash in "kb/s". And therefore GTK will reserve space for e.g. 10 wide chars while the label displays some small chars and thus the label is much wider than necessary.

Reply:
Should this be reworked into the dash approach then? (ref: https://github.com/SoongNoonien/mpdevil/pull/35#discussion_r672515416)

Reply:
Yes I think so. Reading this label and displaying dashes when there is no actual info about the bitrate should be the best solution.